pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -1852,6 +1852,7 @@ func getGatewayClasses(r GatewayResources) map[string]k8s.GatewayController {
 		obj.Status.(*kstatus.WrappedStatus).Mutate(func(s config.Status) config.Status {
 			gcs := s.(*k8s.GatewayClassStatus)
 			*gcs = GetClassStatus(gcs, obj.Generation)
+			gcs.SupportedFeatures = supportedFeatures
```

Comment:
Not sure why we need it.

Reply:
https://github.com/kubernetes-sigs/gateway-api/issues/2162